app/src/main/java/org/simple/clinic/medicalhistory/ui/MedicalHistoryAnswerChip.kt

```diff
@@ -0,0 +1,80 @@
+package org.simple.clinic.medicalhistory.ui
+
+import androidx.compose.animation.animateColorAsState
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.ChipDefaults
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.FilterChip
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import org.simple.clinic.R
+import org.simple.clinic.medicalhistory.Answer
+
+@Composable
+fun AnswerChipsGroup(
```

Comment:
Non-blocking… but I think calling this "MedicalHistoryQuestionOptions" or something similar may point more clearly to the intention. Because "AnswerChipsGroup" hints an arbitrary number of chips, but when it's a question option, we know it's linked to the enum on the server